serving/samples/helloworld-shell/invoke.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	out, err := exec.Command("/bin/sh", "script.sh").Output()
```

Comment:
Output() discards stderr, is that intentional?

Reply:
I am not sure. I assumed stderr would show up in the logs. In which case, yes I prefer for errors to not be returned to the user.

Reply:
Stderr from the subprocess will only appear in actual stderr if it's redirected explicitly.

cmd := exec.CommandContext(...)
cmd.Stderr = os.Stderr
...

Reply:
Done, I confirm that stderr lands in the error logs